independent-publisher-connectors/Clockify/apiDefinition.swagger.json

```diff
@@ -72,18 +214,105 @@
           "200": {
             "description": "Clients List",
             "schema": {
-              "properties": {
-                "id": {
```

Comment:
The type has been changed, can you please confirm that the existing flow will not fail.

Reply:
<img width="450" alt="image-20231127120424043" src="https://github.com/microsoft/PowerPlatformConnectors/assets/897323/b91fa2a9-35fc-47d9-8cdc-988c79398e27">
<img width="1011" alt="image-20231127115710474" src="https://github.com/microsoft/PowerPlatformConnectors/assets/897323/3c01a1ad-9687-45fd-99a4-d5a054c8013f">